### PR TITLE
🚚 Move content from index.markdown

### PR DESCRIPTION
**Pull request by shnizzedy**
## Resolves

Related to #19 by @gkiar 
Related to #15 by @XinhuiLi 

## Description

#19 added [`index.md`](https://github.com/nmind/nmind.github.io/blob/a16050df091f964cb260b2f54a210e4848617861/index.md), so we have both that file and [`index.markdown`](https://github.com/nmind/nmind.github.io/blob/a16050df091f964cb260b2f54a210e4848617861/index.markdown) in the root of the tree: 
`index.md` and `index.markdown`

Most of `index.markdown` was already incorporated into `index.md` or other files in #19. This PR relocates the rest of its contents and removes the extra index file and disables the "coming soon" links.

---

I went back and forth on sequence and horizontal alignment for "Current Members" and "Governance". I wouldn't be salty if we want something different there. My thinking for this layout:

* Thirds look nice except there's only 2 of them.
* Governance doesn't have anything yet, so I thought it could go after the section with content.
* "Governance: Coming Soon" left-aligned looked almost unformatted to me.
* The list of current members centered also looked almost unformatted to me.

Of course, the list could also become an unordered list, but I think that's a separate PR from this one.

### /about
/about

### /community_governance
/community_governance

### /certification#certification-portal
/certification#certification-portal

### CODE REVIEW

File: index.markdown [hunk 1]
```diff
@@ -1,60 +0,0 @@
----
-# Feel free to add content and custom Front Matter to this file.
-# To modify the layout, see https://jekyllrb.com/docs/themes/#overriding-theme-defaults
-
-layout: splash
-header:
-  overlay_color: "#5e616c"
-excerpt: >
-  NMIND is a collaborative dedicated to accelerating scientific discovery in neuroimaging research that was formed in 2020 as a grassroots initiative - responding to the growing frustration about redundancies in effort, continuing questions about the reproducibility of code by independent teams, and challenges arising in the sharing of code between teams due to lack of consistent nomenclature and conventions.
```
Comment (shnizzedy) [inline index.markdown:9]: In https://github.com/nmind/nmind.github.io/blob/4bdcb184616a909f36a4cade129d4e58c8998859/index.md#L12

File: index.markdown [hunk 2]
```diff
@@ -1,60 +0,0 @@
...
-guiding_principals_header:
-  - excerpt: "## Guiding Principles
-
-The NMIND solution has three guiding principles: Alignment, Certification, & Engagement."
```
Comment (shnizzedy) [inline index.markdown:13]: In https://github.com/nmind/nmind.github.io/blob/4bdcb184616a909f36a4cade129d4e58c8998859/index.md#L16-L28

File: index.markdown [hunk 3]
```diff
@@ -1,60 +0,0 @@
...
-
-Development and adoption of standards for critical components of data analysis pipelines. These include, but are not limited to:
-
-- **Coding** standards to promote the adoption of industry standards in academic software, and facilitate code review, collaboration and reuse.
-
-- **Testing** standards to promote reproducibility and efficiency across differing computational environments and use cases.
-
-- **Data performance** standards to promote reproducibility and efficiency of performance across diverse benchmark datasets (e.g. data quality, developmental status, species, clinical populations).
-
-- **Terminology** standards to promote reliable and efficient communication across data analysts and packages."
```
Comment (shnizzedy) [inline index.markdown:25]: In https://github.com/nmind/nmind.github.io/blob/4bdcb184616a909f36a4cade129d4e58c8998859/alignment.md#L6-L36

File: index.markdown [hunk 4]
```diff
@@ -1,60 +0,0 @@
...
-  - excerpt: "### 2. Certification
-
-Accessible and automated mechanisms for certifying the compliance of tools with the NMIND standards. Tool contributors will be able to interact with these mechanisms through a web-based public interface and programmatic toolkits, each facilitating the logging, debugging, and communication of their status."
```
Comment (shnizzedy) [inline index.markdown:28]: In https://github.com/nmind/nmind.github.io/blob/4bdcb184616a909f36a4cade129d4e58c8998859/certification.md#L6-L30

File: index.markdown [hunk 5]
```diff
@@ -1,60 +0,0 @@
...
-NMIND has the following GitHub projects
-
-* [nmind.github.io](https://github.com/nmind/nmind.github.io) - The backing project for this website
-
-* [coding-standards-certification](https://github.com/nmind/coding-standards-certification)
-
-* [hackathon2021](https://github.com/nmind/hackathon2021) - Issues and notes for the May 2021 NMinD hackathon
-
-* [nmind-coding-standards-action](https://github.com/nmind/nmind-coding-standards-action)
-"
```
Comment (shnizzedy) [inline index.markdown:50]: In https://github.com/nmind/nmind.github.io/blob/4bdcb184616a909f36a4cade129d4e58c8998859/engagement.md#L6-L37

File: index.markdown [hunk 6]
```diff
@@ -1,60 +0,0 @@
...
-  - excerpt: "## Current Members
-
-Michael Milham, Damien Fair, Russell Poldrack, Theodore Satterthwaite, Satrajit Ghosh, Greg Kiar, Eric Earl, Chris Markiewicz, Steven Giavasis, Eric Feczko, Oscar Esteban, Mathias Goncalves, Jon Clucas, Teresa George, Xinhui Li, Amanda Rueter, Amy Gutierrez, Anders Perrone, Audrey Houghton, Benjamin Kay, Dorota Jarecka, Gregory Conan, Hecheng Jin, Kathy Snider, Lucille Moore, Robert Hermosillo, Thomas Madison
-"
```
Comment (shnizzedy) [inline index.markdown:54]: In https://github.com/nmind/nmind.github.io/blob/4bdcb184616a909f36a4cade129d4e58c8998859/community_governance.md#L16-L19

**Comment by gkiar** [comment]
Awesome, thanks @shnizzedy !

**Pull request merged**